docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -2,8 +2,14 @@
 [role="xpack"]
 == Rule exceptions and value lists
 
-To prevent the creation of unwanted alerts, you can add exceptions to detection
-rules. Exceptions contain the source event conditions that determine when
+To prevent the creation of unwanted alerts, you can add exceptions to these
+detection rule types:
+
+* Custom query
+* EQL
+* Threat match
```

Comment:
@DonNateR @jmikell821 We recently renamed `Threat Match` to `Indicator Match`. I think you're already aware, but leaving this as a reminder. :)